<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usrSetting</class>
 <widget class="QWidget" name="usrSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="chooseIconBtn">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#chooseIconBtn {
    border-radius: 6px;
    border:2px solid rgb(20,196,188);
background-color: rgb(20,196,188);
    color:  rgb(255, 255, 255);
}
 
QPushButton#chooseIconBtn:hover {
    border-radius: 6px;
    color: rgb(20,196,188);
    background-color: rgb(255,255,255);
  border:2px solid rgb(20,196,188);
    border-color: rgb(20,196,188);
}
 
QPushButton#chooseIconBtn:pressed {
    border: 0px;
    background-color: #ffffff;
    border-radius: 6px;
}
 
QPushButton#chooseIconBtn:disabled {
    border: 0px;
    border-radius: 6px;
    border: 1px solid #46bcff;
    opacity: 0.4;
}</string>
     </property>
     <property name="text">
      <string>Select an icon from this computer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="changeInformationBtn">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#changeInformationBtn {
    border-radius: 6px;
    border:2px solid rgb(20,196,188);
background-color: rgb(20,196,188);
    color:  rgb(255, 255, 255);
}
 
QPushButton#changeInformationBtn:hover {
    border-radius: 6px;
    color: rgb(20,196,188);
    background-color: rgb(255,255,255);
  border:2px solid rgb(20,196,188);
    border-color: rgb(20,196,188);
}
 
QPushButton#changeInformation:pressed {
    border: 0px;
    background-color: #ffffff;
    border-radius: 6px;
}
 
QPushButton#changeInformation:disabled {
    border: 0px;
    border-radius: 6px;
    border: 1px solid #46bcff;
    opacity: 0.4;
}</string>
     </property>
     <property name="text">
      <string>Edit your
 information</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="confirmChangeInfoBtn">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#changeInformationBtn {
    border-radius: 6px;
    border:2px solid rgb(20,196,188);
background-color: rgb(20,196,188);
    color:  rgb(255, 255, 255);
}
 
QPushButton#changeInformationBtn:hover {
    border-radius: 6px;
    color: rgb(20,196,188);
    background-color: rgb(255,255,255);
  border:2px solid rgb(20,196,188);
    border-color: rgb(20,196,188);
}
 
QPushButton#changeInformation:pressed {
    border: 0px;
    background-color: #ffffff;
    border-radius: 6px;
}
 
QPushButton#changeInformation:disabled {
    border: 0px;
    border-radius: 6px;
    border: 1px solid #46bcff;
    opacity: 0.4;
}</string>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="informationTextEdit">
        <property name="styleSheet">
         <string notr="true">QTextEdit#informationTextEdit{
color: rgb(147, 147, 147);
                            	padding-left:6px;

    border: 1px solid rgb(20,196,188);
border-radius: 8px;
    background-color: #ffffff;
};</string>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="filenameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="uploadIconBtn">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Uplaod</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="iconBtn">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#iconBtn {
    border-radius: 6px;
    border:2px solid rgb(20,196,188);
background-color: rgb(20,196,188);
    color:  rgb(255, 255, 255);
}
 
QPushButton#iconBtn:hover {
    border-radius: 6px;
    color: rgb(20,196,188);
    background-color: rgb(255,255,255);
  border:2px solid rgb(20,196,188);
    border-color: rgb(20,196,188);
}
 
QPushButton#iconBtn:pressed {
    border: 0px;
    background-color: #ffffff;
    border-radius: 6px;
}
 
QPushButton#iconBtn:disabled {
    border: 0px;
    border-radius: 6px;
    border: 1px solid #46bcff;
    opacity: 0.4;
}</string>
     </property>
     <property name="text">
      <string>Upload Icon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
